amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/TableCreatorCallback.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Amazon Software License (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://aws.amazon.com/asl/
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.kinesis.leases;
+
+import software.amazon.kinesis.leases.dynamodb.TableCreatorCallbackInput;
+
+/**
+ * Callback interface for interacting with the DynamoDB lease table post creation.
+ */
+public interface TableCreatorCallback {
```

Comment:
I would annotate this with `@FunctionalInterface` if you intend it to be used with a lambda.